models/project/src/mc-project/applied-action-right/generator.ts

```diff
@@ -0,0 +1,13 @@
+import { fake, Generator } from '@commercetools-test-data/core';
+import { TAppliedActionRight } from './types';
+
+const generator = Generator<TAppliedActionRight>({
+  fields: {
+    // TODO: use more realistic values for action rights
+    group: fake((f) => f.animal.type()),
+    name: fake((f) => f.animal.type()),
+    value: fake(() => true),
+  },
+});
```

Comment:
We can use real names like this:

```suggestion
const fakeActionRightNames = [
  'canAddPrices',
  'canEditPrices',
  'canAddProducts',
  'canPublishProducts',
  'canCreateReturns',
  'canSendQuote',
];

const generator = Generator<TAppliedActionRight>({
  fields: {
    name: fake((f) => f.helpers.arrayElement(fakeActionRightNames)),
    value: fake(() => true),
  },
});
```